ide/css.editor/test/unit/data/testfiles/netbeans.css.formatted

```diff
@@ -515,7 +515,7 @@ img.right {
 }
 /* -------------------------- */
 
-/* ----------- Docs & Suport spec styles -------------*/
+/* ----------- Docs & Support spec styles -------------*/
```

Comment:
You're changing a test file, did you test the module afterwards?

Reply:
dependings are changed at 6bf545f6